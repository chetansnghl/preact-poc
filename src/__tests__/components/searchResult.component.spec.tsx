import { h } from 'preact';
import { mount, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-preact-pure';
import SearchResultComponent from './../../components/searchResult.component';
import holidayJson from '../../../__mocks__/hotels.json';

configure({ adapter: new Adapter })

describe('SelectComponent', () => {
    it('should render properly without holidays', async () => {
        const search_result_component = mount(<SearchResultComponent holidays={[]} />)
        expect(search_result_component).toMatchSnapshot();
    })

    it('should render properly', async () => {
        const search_result_component = mount(<SearchResultComponent holidays={holidayJson.holidays} />)
        expect(search_result_component).toMatchSnapshot();
    })    
})