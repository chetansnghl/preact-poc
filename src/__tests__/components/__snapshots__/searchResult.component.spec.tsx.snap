// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectComponent should render properly 1`] = `ReactWrapper {}`;

exports[`SelectComponent should render properly without holidays 1`] = `ReactWrapper {}`;
